sbi/neural_nets/estimators/base.py

```diff
@@ -126,7 +126,7 @@ class ConditionalDensityEstimator(ConditionalEstimator):
     Note:
         We assume that the input to the density estimator is a tensor of shape
         (sample_dim, batch_dim, *input_shape), where input_shape is the dimensionality
-        of the input. The condition is a tensor of shape (batch_size, *condition_shape),
+        of the input. The condition is a tensor of shape (batch_dim, *condition_shape),
```

Comment:
can this be a Shape? or just an int? I think `batch_size` implies it being an int